rclcpp/test/test_intra_process_manager.cpp

```diff
@@ -43,6 +44,13 @@ class Publisher
     return mock_queue_size;
   }
 
+  bool
+  operator==(const rmw_gid_t * gid) const
```

Comment:
No, the method is required to exist on this "mock" class in order to compile and test the other parts of the intra process manager.